econml/tests/test_federated_learning.py

```diff
@@ -150,9 +153,12 @@ def test_splitting_works(self):
                                 with self.subTest(d_x=d_x, d_t=d_t, d_y=d_y,
                                                   weights=(weights is not None), fw=(freq_weights is not None),
                                                   cov_type=cov_type):
-                                    est_all = LinearDML(model_t=t_model, model_y=y_model, linear_first_stages=False)
-                                    est_h1 = LinearDML(model_t=t_model, model_y=y_model, linear_first_stages=False)
-                                    est_h2 = LinearDML(model_t=t_model, model_y=y_model, linear_first_stages=False)
+                                    est_all = LinearDML(model_t=t_model, model_y=y_model, linear_first_stages=False,
+                                                        enable_federation=True)
+                                    est_h1 = LinearDML(model_t=t_model, model_y=y_model, linear_first_stages=False,
```

Comment:
Is there any reason to keep linear_first_stages=False around? Seems like we could start removing its presence from our tests (it is present in other tests as well e.g. test_refit)

Reply:
My thinking was that it's deprecated but not removed I'd leave it in place, but since it's completely ignored even when specified I guess there's no benefit to this, so I'll remove it from tests as well (not just user-facing documentation).